src/types/indexedDocument.type.ts

```diff
@@ -0,0 +1,7 @@
+export interface IndexedDocument {
+  author: string // DID
+  id: string // StreamID
+  parent_author: string // DID
```

Comment:
If the `parent_author` is pulled from the controller manifest of each `TileDocument` then this works, but in most cases we are not going to store the author DID in the subgraph information or even root document.
```javascript
{
    controllers: ["did:3:2341"],  /*<-- We should be able to pull the parent_author from here. 
    Maybe make two types? 
    One for raw data coming from Ceramic and another for what gets put in the database (and often used elsewhere)?*/
},
```